posthog/migrations/0922_cohortcalculationhistory_cohort_started_idx.py

```diff
@@ -0,0 +1,27 @@
+from django.contrib.postgres.operations import AddIndexConcurrently
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    """
+    Add index to speed up CohortViewSet.get_queryset() subquery that fetches
+    the most recent error_code per cohort. Without this index, each subquery
+    requires an in-memory sort (~0.9ms per cohort). With ~500 cohorts, that
+    adds ~450ms to the cohort list API response.
+    """
```

Comment:
I ran some test queries in Metabase to decide whether or not the index was necessary.